### PR TITLE
API getlistuser, insertuser, updateuser  by two way

### PR DESCRIPTION
**Pull request by WindyPT259**
API in api_users_routes.py file

### CODE REVIEW

File: src/controllers/api_user_controller.py [hunk 1]
```diff
@@ -79,21 +97,151 @@ def inser_user():
...
+    return jsonify(results)
+
+### ------API get data using nativeQuery------------------------------------------------------########
+
+
+# GET LIST USER
+def get_list_native_query():
+    try:
+        query = text(
+            "SELECT column_name FROM information_schema.columns WHERE table_name = 'user' AND table_schema = 'sampleapp'")
```
Comment (hvlinhTMA) [inline src/controllers/api_user_controller.py]: What is the table_schema = 'sampleapp'?
With other env, example production env has database name is "ABC", staging env is "ABC_staging", what happen?

File: src/controllers/api_user_controller.py [hunk 2]
```diff
@@ -79,21 +97,151 @@ def inser_user():
...
+        results = {
+            "status_code": 500,
+            "success": False,
+            "message": str(e)
+        }
+    return jsonify(results)
+
+
+# INSERT USER
+def insert_user_native_query():
```
Comment (hvlinhTMA) [inline src/controllers/api_user_controller.py:185]: what happen when user insert was existing on db?

File: src/controllers/api_user_controller.py [hunk 3]
```diff
@@ -79,21 +97,151 @@ def inser_user():
...
+        results = {
+            "status_code": 500,
+            "success": False,
+            "message": str(e)
+        }
+    return jsonify(results)
+
+
+# UPDATE USER
+def update_user_native_query(user_id):
```
Comment (hvlinhTMA) [inline src/controllers/api_user_controller.py:236]: what happen when user not existing on db?

**Review by hvlinhTMA — CHANGES_REQUESTED** [review]

File: src/controllers/api_user_controller.py (hunk 1 above)
Comment (WindyPT259) [reply src/controllers/api_user_controller.py]: - sampleapp: name of my database.
- With other env, I will pass value in this, not hardcode.

File: src/controllers/api_user_controller.py (hunk 2 above)
Comment (WindyPT259) [reply src/controllers/api_user_controller.py:185]: when user insert was existed in db, request will return error from Exception and statuscode 500

Comment (hvlinhTMA) [reply src/controllers/api_user_controller.py:185]: it should be handle api's response 400 bad request with message "user already existing.".

**Pull request closed without merging**